governance.md

```diff
@@ -0,0 +1,101 @@
+## Mission & Scope
+
+NRE Labs grew out of the experiences of working network engineers who were quick to see the value of embracing infrastructure automation for the purpose of network reliability, but were challenged by the learning curve of doing so.
+
+Many automation tools were initially designed for compute and storage automation; in addition much of the work in this area has been done in open source projects, with varying degrees of commercialization. In parallel, the “SDN revolution” initially produced a widely held belief that network engineers would need to become coders or become obsolete.
+
+The group of engineers who started NRE Labs believes strongly that expecting an entire profession to start over and learn programming from first principles is most likely to result in a general failure to adopt network automation at all, even as networking itself becomes ever more important in the basic functioning of distributed, multicloud operations.
+
+### Mission
+
+NRE Labs has been an open source project from its inception--both the platform itself and all the lessons. The NRE Labs community seeks to ensure that anyone anywhere can quickly learn key automation concepts using whatever network equipment and tools they have available. Thus we encourage contributions from as broad a swath of the networking and automation communities as possible.  Anyone can contribute to any aspect of the project, according to the rules laid out in this governance document.
+
+We resolve to provide a platform for education in infrastructure automation tools and techniques for network engineers that can immediately be applied to real-world daily tasks.
+
+We encourage lessons of all types relating to network reliability engineering and infrastructure automation. All lessons should be concise and practical, focused on the mastery of a specific automation task.
+
```

Comment:
Question : should the site/lessons limit to education about publicly available (i.e. open source) tools and techniques or should it be broader (incl. proprietary products) ?

Reply:
FAQ - clarify in contribution docs